packages/system/src/types.ts

```diff
@@ -126,7 +126,7 @@ declare type SynthesizedPath<T extends {}> = {
 export type ThemeNamespaceValue<
   K extends string,
   T extends ITheme,
-> = SynthesizedPath<T[K]>
+> = SynthesizedPath<T[K]> | number | string | false
```

Comment:
I don't think the fix is correct. `ThemeNamespaceValue` means a value in the theme namespace, nothing else. However, I see what you want to fix, but I think it is not the right place to do it.

Reply:
Thank you for your review.
Sure, my fix wasn't fit for name.

Read the code again and investigate the appropriate corrections.

Reply:
1st, I reverted prev changes, and changed SystemProp insted of `ThemeNamespaeceValue` for fix utility props. I think this type name means utility props, but what do you think?

- ddf9537

2nd, above changes is not cover same problem about each themeGetter. so I changed `ThemeGetter` for it.

- 09ea4eb

３rd, the first change on SystemProps found an issue where `col` and `row` props didn't support false, so we fixed each one.

- b46291d
- 5375b4c